images/debian.yaml

```diff
@@ -1107,7 +1122,7 @@ actions:
     # Enable networkd socket
     systemctl enable systemd-networkd.socket
   types:
-  - containers
```

Comment:
@monstermunchkin is "containers" valid at any time? If not should the template validation not have failed the image build entirely with this old value?

Reply:
It's invalid, and we don't have template validation yet.

Reply:
Oh :( I created https://github.com/lxc/distrobuilder/issues/614 for that :)